apps/condo/domains/user/oidc/adapter/OidcModelClientAdapter.js

```diff
@@ -184,7 +184,7 @@ class OidcModelClientAdapter {
      *
      */
     async find (id) {
-        const item = await OidcClient.getOne(this.context, { clientId: id })
+        const item = await OidcClient.getOne(this.context, { clientId: id, isEnabled: true, deletedAt: null })
```

Comment:
Should we send an error about disabled client?

Reply:
It depends. Disabled client for OIDC is similar to no client, since it's not ready for use. Or disabled client is ready-to-use, but just turned off